Possible flaky failure in testChemistry

Consider if we actually need this test. If we do, make the DTD available locally.


```
    [junit] Testcase: testChemistry(nu.xom.tests.BuilderTest):	Caused an ERROR
    [junit] The markup declarations contained or pointed to by the document type declaration must be well-formed. at line 1, column 2
    [junit] nu.xom.ParsingException: The markup declarations contained or pointed to by the document type declaration must be well-formed. at line 1, column 2 in http://www.rsc.org/dtds/entities/rsc_x.ent
    [junit] 	at nu.xom.Builder.build(Unknown Source)
    [junit] 	at nu.xom.Builder.build(Unknown Source)
    [junit] 	at nu.xom.tests.BuilderTest.testChemistry(Unknown Source)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    [junit] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    [junit] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    [junit] Caused by: org.xml.sax.SAXParseExceptionpublicId: -//RSC//RSC characters version 1.0//EN; systemId: http://www.rsc.org/dtds/entities/rsc_x.ent; lineNumber: 1; columnNumber: 2; The markup declarations contained or pointed to by the document type declaration must be well-formed.
    [junit] 	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source)
    [junit] 
```

--- a/src/nu/xom/tests/BuilderTest.java
+++ b/src/nu/xom/tests/BuilderTest.java
@@ -83,7 +83,7 @@ import nu.xom.XMLException;
  * </p>
  * 
  * @author Elliotte Rusty Harold
- * @version 1.4.6
+ * @version 1.5.0
  *
  */
 public class BuilderTest extends XOMTestCase {
@@ -3585,24 +3585,9 @@ public class BuilderTest extends XOMTestCase {
             // Here we expect that because there's no Locator
             // there's no base URL. As a result, the DTD is looked for in the
             // current working directory, which is the wrong place. 
-            // Therefore this IOException is thrown. 
+            // Therefore, this IOException is thrown. 
         }
         
-    }
-    
-    public void testChemistry() throws ValidityException, ParsingException, IOException {
-      String doc = "<!DOCTYPE article PUBLIC \"FOO\" \"http://www.rsc.org/dtds/rscart37.dtd\"><root/>";   
-      StringReader s = new StringReader(doc);
-      try {
-          builder.build(s);
-      } catch (IOException ex) {
-          if (CITestUtil.shouldIgnore(ex)) {
-              // Skip test if network is unavailable in CI
-              systemErr.println("Skipping testChemistry: network unavailable");
-              return;
-          }
-          throw ex;
-      }
     }
     
     public void testMisbehavingParserInternalDTDSubset() throws IOException {

